Include/iterobject.h

```diff
@@ -7,6 +7,9 @@ extern "C" {
 
 PyAPI_DATA(PyTypeObject) PySeqIter_Type;
 PyAPI_DATA(PyTypeObject) PyCallIter_Type;
+#ifdef Py_BUILD_CORE
+extern PyTypeObject _PyAnextAwaitable_Type;
+#endif
```

Comment:
This can be moved inside _PyTypes_Init(). I tried to mimick other header files which defines all types.